docs/source/examples/auto-failover.rst

```diff
@@ -3,29 +3,28 @@
 Auto-provisioning GPUs
 ==========================
 
+SkyPilot comes with an auto-failover provisioner, which
+**automatically retries provisioning** a cluster in different regions (or
+clouds) if the requested resources cannot be provisioned.
+Such failures can happen for a variety of reasons:
+
+- Out of capacity (in a region or a cloud)
+- Out of quotas (in a region or a cloud)
```

Comment:
What's quotas? Consider:
`Out of quotas (you have reached your allowed limit in a region or cloud)`

Reply:
See the other comment. I think this is a bit inaccurate, too, since the reached quota can be #vcpus, #ips_in_region, #instance_for_type, etc.